src/aura/dataTable/dataTableController.js

```diff
@@ -10,20 +10,21 @@
 	        ]);
 
 		// Firt load, load all Accounts
-		helper.loadAllAccounts(component);
+		helper.loadAccounts(component);
 	},
-	viewContactsFromAccount: function(component, event, helper) {
-		console.log('viewContactsFromAccount');
-		var accountId = event.target.dataset.account;
+	viewContactsFromAccount: function(component, event, helper) {		
+		var accountId = event.target.dataset.account,
+			appEvent = $A.get("e.c:searchContacts");
```

Comment:
be aware of indentation

Reply:
<img width="681" alt="screen shot 2017-11-13 at 11 20 11 am" src="https://user-images.githubusercontent.com/33488023/32730074-a960f27e-c864-11e7-825f-1d65b4b3e94b.png">
Maybe its an IDE issue?

Reply:
may be a combination of spaces and tabs, the atom is fixing it visually for you but it seems its wrong. NW